WooCommerce/Classes/ViewRelated/Notifications/ReviewViewModel.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import Yosemite
+
+final class ReviewViewModel {
+    private let review: ProductReview
+    private let product: Product?
+
+    let notIcon: String = "\u{f300}"
```

Comment:
I think it is a font name. I guess that's why the outlet in the cell was named like that (I lifted the cell from the previous notifications, and then extracted a view model out of it, so I left the names unchanged)

Reply:
Yep. The variable name is an indicator that we're using the Noticons font symbols instead of Gridicons.

<img width="512" alt="Screen Shot 2019-09-04 at 3 22 19 PM" src="https://user-images.githubusercontent.com/1062444/64290279-74b3e200-cf2b-11e9-9e06-0db9bcec4219.png">